src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -73,93 +180,92 @@ def generate_WDB_1_default_config(self, filename):
 
     # Create the actual config file
     config = {
-        'palRecord': pal_record_0,
-        'registrationRequest': device_b,
-        'conditionalRegistrationData': conditionals,
-        'fakePalDatabaseInfo': fake_pal_database_config
+        'palIds': [pal_ids_a, pal_ids_b],
+        'registrationRequests': [device_a, device_b],
+        'conditionalRegistrationData': conditionals
+        # TODO
+        # Need to add data base configurations.
     }
     writeConfig(filename, config)
 
   @configurable_testcase(generate_WDB_1_default_config)
   def test_WINNF_FT_S_WDB_1(self, config_filename):
     """PAL Database Update."""
 
+    # Load the configuration file
     config = loadConfig(config_filename)
 
-    # Step 1: Register CBSD C to the SAS UUT.
-    cbsd_ids = self.assertRegistered([config['registrationRequest']],
-                        config['conditionalRegistrationData'])
-
-    # Step 2: Request creation of PPA 
-    pal_id = config['palRecord']['palId']
-    ppa_creation_request = {
-            "cbsdIds": cbsd_ids,
-            "palIds": [pal_id]
-        }
-    ppa_id = self._sas_admin.TriggerPpaCreation(ppa_creation_request)
+    # Step 1: Register device(s) 'C' with conditional registration data
+    cbsd_ids = self.assertRegistered(config['registrationRequests'],
+                                     config['conditionalRegistrationData'])
 
-    # Validate PPA creation request is failed.
-    self.assertEqual(ppa_id, None)
+    # Step 2: Request creation of PPA(s).
+    # Forming of PPA creation request(s)
+    ppa_creation_requests = []
+    for pal_ids in config['palIds']:
+      ppa_creation_requests.append({
+        "cbsdIds": cbsd_ids,
```

Comment:
There should be a separate set of CBSDs for each PPA creation request. As the note at the beginning of the test says, the sub-tests should *not* interact with one another. (Moreover, a single CBSD may not belong to more than one PPA, so here the SAS may pass for the wrong reason.)